File: twitter/src/main/java/com/twitterconnect/controller/WebEndPoint.java
package com.twitterconnect.controller;

/**
 * Created by amit on 4/2/17.
 */

public class WebEndPoint {
    public static final String USER_ID_KEY = "user_id";
    private static final String BASE_URL = "https://api.twitter.com/";
    protected static final String VERSION = "1.1/";

    protected static final String PROFILE = "/" + VERSION + "users/show.json";
    protected static final String FRIENDS = "/" + VERSION + "friends/list.json";

    protected static int PROFILE_TASK_ID = 1;
}

File: twitter/src/main/kotlin/com/twitterconnect/controller/WebEndPoint.kt
package com.twitterconnect.controller

/**
 * Created by amit on 4/2/17.
 */

object WebEndPoint {
    val USER_ID_KEY = "user_id"
    private val BASE_URL = "https://api.twitter.com/"
    const val VERSION = "1.1/"

    const val PROFILE = "/" + VERSION + "users/show.json"
    const val FRIENDS = "/" + VERSION + "friends/list.json"

    internal var PROFILE_TASK_ID = 1
}
